packages/fxa-content-server/app/scripts/models/account.js

```diff
@@ -367,6 +367,34 @@ const Account = Backbone.Model.extend(
       });
     },
 
+    /**
+     * Fetches account details from GET /account, for use by the settings views.
+     * Caches its own result, because it's not intended for that endpoint to be
+     * called repeatedly like some of the polling methods are. If you're really,
+     * really sure you need to, pass the `force` option to force a clean request.
```

Comment:
If the user's card expires, how would caching invalidation happen?

Reply:
It doesn't. I'm operating on the basis that:

1. Users don't tend to have long-lived sessions clicking around the settings view in FxA.

2. The message is informative rather than some kind of mandatory error, so it doesn't matter in the cases where it doesn't immediately pop up on someone's view the moment a card expires.

3. We don't really want to hit the auth server for this endpoint every time e.g. navigation between settings panels occurs.

And of course, it's not a cache in any real sense, a page refresh or a new session is all that's required to force the call to happen again.

But any or all of those assumption might be wrong.